faraday/deviceconfiguration.py

```diff
@@ -230,7 +236,9 @@ def configureDeviceConfiguration(args, deviceConfigurationConfigPath, faradayCon
     # Create configuration boot bitmask integer
     bootmask = [0] * 8
     redledtx = fconfig.get('BASIC', 'REDLEDTX')
+    greenledrx = fconfig.get('BASIC', 'GREENLEDRX')
     unitconfigured = fconfig.get('BASIC', 'UNITCONFIGURED')
+    bootmask[2] = greenledrx
```

Comment:
This is incorrect and should be `bootmask[5]`.

The root cause of this error is confusing eightBitListToInt() ordering functionality but this should be clarified/updated in a future PR.